File: src/tasks/cli/command/complete/CompleteArguments.java
package tasks.cli.command.complete;

import omnia.data.structure.List;
import tasks.cli.arg.SimpleArguments;
import tasks.model.Task;

public final class CompleteArguments extends SimpleArguments {
  CompleteArguments(List<Task> tasks) {
    super(tasks);
  }
}

File: cli/src/me/strangepan/tasks/cli/feature/complete/CompleteArguments.kt
package me.strangepan.tasks.cli.feature.complete

import omnia.data.structure.List
import me.strangepan.tasks.cli.command.common.simple.SimpleArguments
import me.strangepan.tasks.engine.model.Task

/** Model for parsed Complete command arguments.  */
class CompleteArguments internal constructor(tasks: List<Task>) : SimpleArguments(tasks)